packages/vincent-tools/vincent-tool-uniswap-swap/package.json

```diff
@@ -5,81 +5,61 @@
     "access": "public"
   },
   "dependencies": {
-    "@lit-protocol/vincent-contracts": "0.1.1-4",
+    "@lit-protocol/vincent-policy-spending-limit": "workspace:*",
     "@lit-protocol/vincent-tool-sdk": "workspace:*",
+    "@uniswap/sdk-core": "^7.7.2",
+    "@uniswap/v3-core": "^1.0.1",
+    "@uniswap/v3-periphery": "^1.4.4",
+    "@uniswap/v3-sdk": "^3.25.2",
     "ethers": "^5.7.2",
-    "tslib": "^2.8.1",
+    "viem": "^2.23.2",
     "zod": "^3.24.1"
   },
   "devDependencies": {
     "@dotenvx/dotenvx": "^1.39.0",
-    "@lit-protocol/auth-helpers": "^7.0.8",
-    "@lit-protocol/constants": "^7.0.8",
-    "@lit-protocol/contracts-sdk": "^7.0.6",
-    "@lit-protocol/lit-auth-client": "^7.0.6",
-    "@lit-protocol/lit-node-client": "^7.0.8",
     "@types/jest": "^29.5.14",
+    "cbor": "^10.0.3",
     "esbuild": "^0.19.11",
     "jest": "^29.7.0",
     "jest-environment-node": "^29.7.0",
     "jsonc-eslint-parser": "^2.4.0",
     "node-fetch": "^2.7.0",
     "ts-jest": "^29.2.6",
-    "ts-node": "^10.9.2",
-    "viem": "^2.23.2"
+    "ts-node": "^10.9.2"
   },
   "type": "commonjs",
   "main": "./dist/src/index.js",
+  "module": "./dist/src/index.js",
   "types": "./dist/src/index.d.ts",
-  "typings": "./dist/src/index.d.ts",
   "files": [
     "dist",
     "!**/*.tsbuildinfo"
   ],
   "scripts": {
-    "test:contract-caller": "npx ts-node tools/scripts/vincent-contract-caller.ts",
-    "test": "jest --testPathIgnorePatterns=test/tool.spec.ts",
-    "test:execute-tools": "jest test/tool.spec.ts --bail 1",
-    "test:spending-limit-error": "jest test/spending-limit-error.spec.ts --bail 1"
+    "test": "jest"
   },
   "nx": {
```

Comment:
🔪 please migrate the nx configuration to `project.json` for consistency with the rest of the repo